tools/codegen/openapi.stainless.yml

```diff
@@ -0,0 +1,220 @@
+# yaml-language-server: $schema=https://app.stainlessapi.com/config-internal.schema.json
```

Comment:
This is a generated file that we can actually edit, correct? It would be good to have some comments or details about this file either here or in the readme

Reply:
This is not a generated file, it's an editable config file.

I'll add some notes in the readme